config/pmd.xml

```diff
@@ -7,9 +7,40 @@
   <description>
     PMD common ruleset for Checkstyle main/test sourcesets
   </description>
+  <rule ref="category/java/errorprone.xml">
+    <!-- That rule is not practical, no options to allow some magic numbers,
+         we will use our implementation. -->
+    <exclude name="AvoidLiteralsInIfCondition"/>
+    <!-- Too many false positives. -->
+    <exclude name="BeanMembersShouldSerialize"/>
+    <!-- We need compare by ref as Tree structure is immutable, we can easily
+         rely on refs. -->
+    <exclude name="CompareObjectsWithEquals"/>
```

Comment:
please remove it from "rulesets/java/design.xml" in this PR.

Looks like all other should be removed from old location too, please do.